airflow/dags/weekly_action_items.py

```diff
@@ -0,0 +1,62 @@
+from datetime import datetime,date
+from airflow.providers.mongo.hooks.mongo import MongoHook
+from airflow.providers.apache.kafka.operators.produce import ProduceToTopicOperator
+from airflow.decorators import dag, task
+from bson import json_util
+import json
+
+@dag(schedule_interval='@daily', start_date=datetime(2024, 1, 1), catchup=False)
+def weekly_action_items():
+
+    @task.python(retries=5)
+    def get_clients():
+        import calendar
+
+        try:
+            hook = MongoHook(mongo_conn_id='metadata')
+            connection = hook.get_conn()
+            clients_collection = connection.metadata.clients
+
+            print(f"Connected to MongoDB - {connection.server_info()}")
+            today = calendar.day_name[date.today().weekday()]
+            result = list(clients_collection.find({"actionsRefreshWeekday": today}, {"tokens": 0, "password": 0, "__v": 0}))
+            output = json.loads(json_util.dumps(result))
+
+            for client in output:
+                client["_id"] = client["_id"]["$oid"]
```

Comment:
Are you sure that mongoose returns the data in this format? I think _id holds the value